uidefinitions/createUiDefinition.json

```diff
@@ -1795,7 +1795,7 @@
                 "visible": "[and(steps('ood').OODSection.deployOOD,equals(int(steps('scheduler').loginNodesSection.initialNodes),0))]",
                 "options": {
                   "icon": "Warning",
-                  "text": "Open OnDemand for Azure CycleCloud requires a minimum of one login node for use as a proxy between Azure CycleCloud and the Open OnDemand web server. Please review selections made under the Slurm Settings menu."
+                  "text": "Open OnDemand requires a minimum of one login node for use as a proxy between Azure CycleCloud and the Open OnDemand web server. Please review selections made under the Slurm Settings menu."
                 }
```

Comment:
Open OnDemand for Azure CycleCloud was intentional - you don't actually need login nodes to work with OOD if you use a different manner of running commands on slurm - for example, often people would install the slurm binaries on the OOD node and have a different slurm.conf for each cluster that they would use.